DEVELOPING_OPAL/demos/src/main/scala/org/opalj/tac/CloseResources.scala

```diff
@@ -6,9 +6,6 @@ import java.net.URL
 
 import scala.jdk.CollectionConverters._
 
-// import org.opalj.log.OPALLogger.info
```

Comment:
Can we avoid the removal of commented out imports? They are sometimes useful to restore necessary imports for commented out code.

Reply:
I checked. Scalafix removes all comments in the imports. If we put the commented out imports below all others, they are kept.

Reply:
I think I've fixed all of these. But there might be other comments we are losing ...